ray_data_eval/image_training/e2e_training/ray_data_e2e_training.py

```diff
@@ -325,40 +327,103 @@ def train(train_dataset, model, criterion, optimizer, epoch, device, args):
 
     i = 0
     end = time.time()
-    for batch in train_dataset.iter_torch_batches(batch_size=args.batch_size):
+
+    def _collate_fn_arr_pil_images_to_tensor(batch:dict) -> dict:
```

Comment:
space after `:`. did you run formatter?

Reply:
Oops, turned out I forgot to install Ruff on my azure machine. Looked like the previous commits passed the push checks but I just reformatted the documents using Ruff.